package.json

```diff
@@ -8,7 +8,8 @@
     "build": "node bin/build.js",
     "start": "npm run clean && npm run build",
     "version": "npm run build && git add -A dist dist/* jquery-src bin",
-    "test": "karma start test/karma.conf.js"
+    "test": "karma start test/karma.conf.js",
+    "test:jasmine": "jasmine test/test.js"
```

Comment:
good call, we really don't need a browser for this, I was just copying from elsewhere.

Any reason not to just make jasmine the default runner and get rid of karma?